### ISSUE TITLE
Parser divergence: multiline strings in JSX attributes

### ISSUE THREAD

**Issue by DanielRosenwasser** [microsoft/typescript-go#125]
When strings are directly used in the right side of a JSX attribute, they may be multiline. [These strings have a different production rule](https://facebook.github.io/jsx/#prod-JSXAttributeValue) in the JSX spec.

```tsx
export let x = <div className="align-top
  align-right" />;
```

**Expected**: No errors
**Actual Behavior**:

```
bar.tsx:1:41 - error TS1002: Unterminated string literal.

1 export let x = <div className="align-top
                                          

bar.tsx:2:14 - error TS1003: Identifier expected.

2   align-right" />
               ~~~~

bar.tsx:2:18 - error TS1002: Unterminated string literal.

2   align-right" />
                   


Found 3 errors in the same file, starting at: bar.tsx:1
```
